### ISSUE TITLE
Signup Mutation throwing error

### ISSUE THREAD

**Issue by Azad99-9** [PalisadoesFoundation/talawa-api#2203]
**Describe the bug**
When trying to signup into the mobile application using the below mutation

```
mutation SignUp($data: UserInput!) {
  signUp(data: $data) {
    appUserProfile{
              adminFor{
                _id
                name
              }
              createdOrganizations{
                _id
                name
                image
                description
                userRegistrationRequired
                creator{
                  _id
                  firstName
                  lastName
                  image
                }
              }
            }
            user{
                _id
                firstName
                lastName
                email
                image
                joinedOrganizations{
                  _id
                  name
                  image
                  description
                  userRegistrationRequired
                  creator{
                    _id
                    firstName
                    lastName
                    image
                  }
                }
                membershipRequests{
                  organization{
                    _id
                    name
                    image
                    description
                    userRegistrationRequired
                    creator{
                      _id
                      firstName
                      lastName
                      image
                    }
                  }
                }
              }
              refreshToken
              accessToken
  }
}

```
I am getting this error
![Screenshot from 2024-04-14 08-35-29](https://github.com/PalisadoesFoundation/talawa-api/assets/120930148/e77cac5e-40e6-41fb-b12f-7a39c7254f7a)

**To Reproduce**

1. Use the above-mentioned mutation
2. Signup with a fresh account

**Expected behavior**
The user should be signed up without any errors.

**Actual behavior**
The account is being created but throwing errors.

**Screenshots**
[Screencast from 14-04-24 07:30:12 PM IST.webm](https://github.com/PalisadoesFoundation/talawa-api/assets/120930148/0ed34e0b-5253-42ce-ae7e-cceaa4f64571)

**Additional details**
This bug is a dependency for the issue in the mobile repo 
https://github.com/PalisadoesFoundation/talawa/issues/2330

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by meetulr** [comment]
I have fixed it, though I have two issues assigned, one is a minor fix, and the PR is ready to be merged #2204.
Assign it to me if this is urgent, I can raise a PR right away.

**Comment by AVtheking** [comment]
I have also fixed it ,if @meetulr  can't be assigned because of the two issues policy then I can solve it .
